pdc/apps/release/tests.py

```diff
@@ -1275,6 +1298,17 @@ def test_patch_integrated_with(self):
         self.assertEqual(response.data.get('integrated_with'), 'release-2.0')
         self.assertNumChanges([1])
 
+    def test_update_sigkey(self):
+        response = self.client.put(self.url,
+                                   {'short': 'product', 'version': '1.0', 'release_type': 'ga',
+                                    'name': 'Our Product', 'sigkey': 'ABCDEF'},
+                                   format='json')
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
```

Comment:
As you update sigkey, why not to verify sigkey's value :)

Reply:
Good point, just change it.